specification/cognitiveservices/data-plane/ContentSafety/preview/2023-11-30-preview/contentsafety.json

```diff
@@ -0,0 +1,3161 @@
+        },
+        "message": {
+          "type": "string",
+          "description": "A human-readable representation of the error."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the error."
+        },
+        "details": {
+          "type": "array",
+          "description": "An array of details about specific errors that led to this reported error.",
+          "items": {
+            "$ref": "#/definitions/Azure.Core.Foundations.Error"
+          },
+          "x-ms-identifiers": []
+        },
+        "innererror": {
+          "$ref": "#/definitions/Azure.Core.Foundations.InnerError",
+          "description": "An object containing more specific information than the current object about the error."
+        }
+      },
+      "required": [
+        "code",
+        "message"
+      ]
+    },
+    "Azure.Core.Foundations.ErrorResponse": {
+      "type": "object",
+      "description": "A response containing error details.",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/Azure.Core.Foundations.Error",
+          "description": "The error object."
+        }
+      },
+      "required": [
+        "error"
+      ]
+    },
+    "Azure.Core.Foundations.InnerError": {
+      "type": "object",
+      "description": "An object containing more specific information about the error. As per Microsoft One API guidelines - https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses.",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "One of a server-defined set of error codes."
+        },
+        "innererror": {
+          "$ref": "#/definitions/Azure.Core.Foundations.InnerError",
+          "description": "Inner error."
+        }
+      }
+    },
+    "DetectUngroundednessGptxResource": {
+      "type": "object",
+      "description": "Connection information for GPT resource.",
+      "properties": {
+        "azureOpenAIEndpoint": {
+          "type": "string",
+          "description": "Endpoint of Azure OpenAI resource."
+        },
+        "deploymentName": {
+          "type": "string",
+          "description": "Deployment name of GPT model."
+        }
+      },
+      "required": [
+        "azureOpenAIEndpoint",
+        "deploymentName"
+      ]
+    },
+    "DetectUngroundednessOptions": {
+      "type": "object",
+      "description": "The request of ungroudedness detection.",
+      "properties": {
+        "domainCategory": {
+          "$ref": "#/definitions/TextDomainCategory",
+          "description": "Domain of the text to be analyzed."
+        },
+        "task": {
+          "$ref": "#/definitions/TextTask",
+          "description": "Task type of the text to be analyzed."
+        },
+        "qaQuery": {
+          "type": "string",
+          "description": "This is optional and only used for QnA or chat task."
```

Comment:
> QnA or chat
Actually, when I read this part, I'm confusing what's the difference between task type QNA and Chat.